doc/engineering-reference/src/surface-heat-balance-manager-processes/outside-surface-heat-balance.tex

```diff
@@ -781,6 +781,11 @@ \subsection{Exterior/External Conduction}\label{exteriorexternal-conduction}
 
 The conduction term, \({q''_{ko}}\) \(_{,}\) can in theory be calculated using a wide variety of heat conduction formulations. Typically in EnergyPlus, the Conduction Transfer Function (CTF) method is used. The available models are described in this section: Conduction Through The Walls.
 
+\subsection{Additional Heat Balance Source}\label{additonal-heat-balance-source}
+
+EnergyPlus also enables importing the pre-calculated results of other heat transfer processeses, such as evaporative cooling envelope. An additional heat source term defined as a surface property would enable the consideration of these processes to be imported as schedules in the exterior surface heat balance calculation in EnergyPlus.
```

Comment:
Is there an equation in the Engineering Reference where this term should be added?